tests/system/test_system.py

```diff
@@ -558,9 +670,16 @@ def test_backup_workflow(self):
         backup_id = "backup_id" + unique_resource_id("_")
         expire_time = datetime.utcnow() + timedelta(days=3)
         expire_time = expire_time.replace(tzinfo=UTC)
+        version_time = datetime.utcnow() - timedelta(seconds=5)
```

Comment:
There is an assumption here that the instance is 5 seconds old. I think this is a fair assumption, but just want to make sure that we know the database that we are backing up is always at least 5 seconds old when we reach this, otherwise we can get flaky tests.